.github/workflows/build-macos.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         # See https://help.github.com/en/articles/workflow-syntax-for-github-actions.
         name: [
           macos-12-xcode-14.2,
+          macos-14-xcode-15.4,
```

Comment:
M1 runner is super expensive, don't know if you want it to be on each PR update...

A manual trigger is probably better, just saying.

Reply:
Hmmm. Expensive in what sense?